apps/web/pages/availability/[schedule].tsx

```diff
@@ -153,7 +154,7 @@ export default function Availability() {
 
   return (
     <Shell
-      backPath="/availability"
+      backPath={fromEventType ? true : "/availability"}
```

Comment:
If we create a new schedule from an event type, the back button should bring the user back to the event type.